src/test/java/org/jabref/logic/importer/OpenDatabaseTest.java

```diff
@@ -28,19 +28,24 @@
     private final Charset defaultEncoding = StandardCharsets.UTF_8;
     private ImportFormatPreferences importFormatPreferences;
     private TimestampPreferences timestampPreferences;
-    private final Path bibNoHeader;
-    private final Path bibWrongHeader;
-    private final Path bibHeader;
-    private final Path bibHeaderAndSignature;
-    private final Path bibEncodingWithoutNewline;
+    private Path bibNoHeader = null;
+    private Path bibWrongHeader = null;
+    private Path bibHeader = null;
+    private Path bibHeaderAndSignature = null;
+    private Path bibEncodingWithoutNewline = null;
     private final FileUpdateMonitor fileMonitor = new DummyFileUpdateMonitor();
 
-    OpenDatabaseTest() throws URISyntaxException {
+    OpenDatabaseTest() {
```

Comment:
Why did you change this? If you ignore the tests if the files are not found, we won't see if anyone accidently deleted the files.

Reply:
Agree with @Siedlerchr : This seems to me an intrinsic part of the test: To see, if the files are there.